src/views/Settings.vue

```diff
@@ -157,4 +181,7 @@ export default defineComponent({
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
   }
-</style>
+  hr {
```

Comment:
I think styling for hr tag is not needed, as we are using apps-theme package.

Reply:
Sir, two hr tags are getting rendered if we remove the styling then the priority is given to the one with the border width of 0 px. Hence, it's not working as the theme styling is not prioritised by the browser.